pyproject.toml

```diff
@@ -169,7 +169,6 @@ markers = "sys_platform != 'darwin' or platform_machine != 'arm64'"
 optional = true
 
 [[tool.poetry.dependencies.pydantic]]
-# FIXME: https://github.com/explosion/spaCy/issues/12034
```

Comment:
I'd keep this comment for future reference (maybe amend with added explanation to remove constraint only after issue is fixed)